CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Join the development
+* Before you join development, please set up the project on your local machine, run it and go through the application completely. Press on any button you can find and see where it leads to. Explore.You'll be more familiar with what is where and might even get some cool ideas on how to improve various aspects of the app.
+* If you would like to work on an issue, drop in a comment at the issue. If it is already assigned to someone, but there is no sign of any work being done, please free to drop in a comment so that the issue can be assigned to you if the previous assignee has dropped it entirely.
+
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, or the official [slack channel](https://slack-ssl-openfoodfacts.herokuapp.com/).
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Check that there are no conflicts and your request passes [Travis](Travis-ci.org) build. Check the log of the pass test if it fails the build.
```

Comment:
Can you make this an absolute link? This does not work as expected when rendered in Markdown.

Reply:
Yeah sorry I didnt noticed it will do so surely